microsoft-edge/devtools-guide-chromium/crash-analyzer/index.md

```diff
@@ -158,13 +158,13 @@ To use the **Crash analyzer** tool in DevTools:
 
    This loads the source maps that correspond to the source modules, and attempts to unminify any stack frames for which JavaScript or TypeScript source content was contained in the source map.
 
-1. If the source content was contained for a frame, click on the frame.  Click through the individual stack frames, to see the lines in your original code that caused the error.
+1. If the source code is available in your source maps for a frame, click on the frame to display the source code.
 
-   That opens that source content in the **Sources** tool and highlights the line in question:
+   The original source code appears in the **Sources** tool and the line in question is highlighted:
 
    ![Unminified code file and line of code accessed by clicking in the right-hand pane of the Crash analyzer tab](./index-images/code-displayed-from-right-pane.png)
 
-1. You can further copy the unminified stack trace by clicking the **Copy** button on the toolbar.  That produces (and copies to the clipboard) a stack trace in the conventional V8 stack trace format, except with resolved function names, source files, and line/column pairs also resolved:
+1. You can further copy the unminified stack trace by clicking the **Copy** button on the toolbar.  That produces (and copies to the clipboard) a stack trace in the conventional stack trace format, except with function names, source files, and line/column pairs that match your original code:
```

Comment:
my comprehension stumbled at "with".  might be a case covered by 
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/u/using-by-using-with
This probably means "added" or "add'l" - might revise to add that, to clarify.

Reply:
in table below, adds the word "resolved", which seems clearer.  consider revising table & this, to match.

Reply:
is button tooltip "Copy", or "Copy unminified stack trace"?
I'm trying to improve wording and make these two wordings the same:
```
1. You can further copy the unminified stack trace by clicking the 
   **Copy** button on the toolbar.  That produces (and copies to 
   the clipboard) a stack trace in the conventional stack trace 
   format, except with function names, source files, and line/column 
   pairs that match your original code:

| The **Copy unminified stack trace** (![the Copy unminified stack trace' icon](./index-images/copy-unminified-stack-trace-icon.png)) 
button | Copies the full, unminified stack trace, as a stack trace 
in the conventional stack trace format, except with resolved 
function names, source file names, and line and column numbers. |
```
@captainbrosset

Reply:
changed both spots to:

the **Copy unminified stack trace** (![the Copy unminified stack trace' icon](./index-images/copy-unminified-stack-trace-icon.png)) button ... produces and copies to the clipboard the full, unminified stack trace, as a stack trace in the conventional stack trace format, except with information that matches your original code, including function names, source file names, and line and column numbers